aptly/publisher/__init__.py

```diff
@@ -186,6 +186,91 @@ def _get_publish(self):
                 return publish
         raise NoSuchPublish("Publish %s does not exist" % self.name)
 
+    def save_publish(self, aptly_dir="/srv/aptly/"):
+        """
+        Serialize publish in YAML
+        """
+        name = self.name.replace('/', '-')
+        file_name = aptly_dir + name + "-" + time.strftime("%Y%m%d%H%M%S")
+        lg.info("Saving publish %s in %s" % (name, file_name))
+        with open(file_name, 'w') as save_file:
+            save_file.write("publish: %s\n" % self.name)
```

Comment:
Instead of writing yaml manually, create dict structure and use `yaml.dump` - http://pyyaml.org/wiki/PyYAMLDocumentation#DumpingYAML